odoo/tools/config.py

```diff
@@ -643,6 +643,8 @@ def _load_cli_options(self, opt):
 
     def _postprocess_options(self):
         self._runtime_options.clear()
+        if self['init'] == ['account']:
```

Comment:
seems like forgotten test code

Reply:
Oupsie, please ignore this review request. Eric just cherry-picked a commit from a WIP branch that fixes the tests checking the balance sheets reports to validate this new one.
That cherry-picked commit will of course be removed. Thanks :)